packages/l/libsdl/xmake.lua

```diff
@@ -61,6 +61,7 @@ package("libsdl")
     add_configs("use_sdlmain", {description = "Use SDL_main entry point", default = true, type = "boolean"})
     if is_plat("linux") then
         add_configs("with_x", {description = "Enables X support (requires it on the system)", default = true, type = "boolean"})
+        add_configs("with_wayland", {description = "Enables Wayland support (requires it on the system)", default = true, type = "boolean"})
```

Comment:
We shouldn't enable x11 and wayland at the same time, it will lead to installing a lot of dependencies. Users don't usually use both at the same time.

If you need wayland, you can enable it yourself, `{configs = {with_wayland = true}}`.

I will disable it by default.

Reply:
I have improved this package. https://github.com/xmake-io/xmake-repo/pull/2563